README.md

```diff
@@ -0,0 +1,45 @@
+# NODEMAILER
+
+Foobar is a Python library for dealing with word pluralization.
+
```

Comment:
Is this relevant?

Reply:
Sorry, it's a markup i used to create readme files and i didn't correct that part